src/controller.cpp

```diff
@@ -58,7 +67,8 @@ geometry_msgs::msg::TwistStamped Controller::computeVelocityCommands(
     optimizer_.evalControl(robot_pose, robot_speed, transformed_plan, goal_checker);
 
   visualize(robot_pose, robot_speed, std::move(transformed_plan));
-  return cmd;
+
+  return stop_robot_ ? geometry_msgs::msg::TwistStamped{} : cmd;
```

Comment:
Why? Remove this and the parameter

Reply:
I wanted to see optimization process with different parameters while robot not moving (so i can see generated trajectories and other stuffs) 
How can i do this differently?

Reply:
Ah, I see, what I do is remap the `cmd_vel` topic to a bogus topic (`fake_commands` or something) so as it produces commands, it goes into a black hole never to be received by the simulator.

Reply:
Oh, sure. Thanks. I'll remove it

Reply:
removed